hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/RecordMigrationInfo.java

```diff
@@ -27,6 +27,14 @@
     private Data key;
```

Comment:
After short discussion with Emin I came to the understanding it isn't needed. The class name is not being send over the wire.